src/library_async.js

```diff
@@ -341,9 +424,11 @@ mergeInto(LibraryManager.library, {
     },
   },
 
+  emscripten_sleep__sig: 'vi', // TODO: others
```

Comment:
Do we need that comment?

Reply:
I think so, we'll need to add sigs for all methods in this file eventually? We depend on that with the new wasm API.

Reply:
I'm adding a better comment for this.

Reply:
What I means is, all library functions could benefit from adding as .sig so this comment could equally well apply to all `__sig` lines in the whole codebase.  

In order words, that TODO to add more __sigs is more of a codebase-wide TODO.. I'm not sure it makes sense to mention it in just one specific place.

Reply:
It is far more urgent here, though: literally it will trap without a sig, when using stack switching.

Reply:
Thats that same for all functions though.  For example, if you try to call `addFunction` with any JS library function that does't have a `__sig` it will crash/trap.

Reply:
(Sorry let me rephrase that: if you try to take the address of a library function within a SIDE_MODULE, and that library function doesn't have a `__sig`, it will fail).

Reply:
Yes, I guess to me this feels more urgent since I think users are more likely to run into it... but maybe that's wrong.

I can remove the TODO if you prefer.

Reply:
Ok, I edited it now to focus on the overall work to get this entire file to work with stack switching. The sigs are just a part of that.